crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -350,7 +356,12 @@ impl AptosDataBuilder {
             })
         };
 
-        crate::logger::set_global_logger(logger.clone());
+        let console_port = if cfg!(disable_tokio_console) {
```

Comment:
where is disable_tokio_console being set?

Reply:
It feels to me like we should be doing the opposite: assume it's always disabled unless someone passes a flag, e.g., `enable_tokio_console`? Or better yet, if we're making the port configurable via the `LoggingConfig`, why not also have a flag in the config? e.g., introduce a new config `TokioConsoleConfig` that contains the port and flag to enable it?

Reply:
I fear that this will remain a dead code until people are introduced to these tools.
I wouldn't add `TokioConsoleConfig`:
a. just for one field Option<u16> 
b. it's intimately related to aptos-logger as both deal with the tracing subsystem.